stats/fly.toml

```diff
@@ -36,3 +36,6 @@ processes = []
   [[services.ports]]
     handlers = ["tls", "http"]
     port = 443
+
+[deploy]
+  release_command = "npm run migrate"
```

Comment:
IMO, the stats REST API server should not run the migration in production, because it does not own the database schema (does not write new data). The schema is owned by spark-observer.